third-party/imgui/realsense_imgui.cpp

```diff
@@ -0,0 +1,469 @@
+#include <realsense_imgui.h>
```

Comment:
Missing copyrights

Reply:
Please also explain (comment in the code) the need of this file

Reply:
added copyrights and explanation for this file